compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -347,6 +347,9 @@ error.invalid.record.rest.descriptor=\
 error.missing.required.record.field=\
   missing non-defaultable required record field ''{0}''
 
+error.invalid.field.for.readonly.record.field=\
+  invalid field; ''readonly'' field expected for ''{0}''
```

Comment:
Shouldn't this be a colon?

Reply:
Fixed, but we seem to have used a mix of `:` and `;` in error messages.